tools/go-generator/cmd/generator/_templates/loader.go.tmpl

```diff
@@ -58,6 +58,17 @@ func getInt32Env(name string) (int32, bool) {
 	return 0, false
 }
 
+// getInt64Env returns the int64 value for an env var and a confirmation
+// if the var exists
+func getInt64Env(name string) (int64, bool) {
+	if val := os.Getenv(name); val != "" {
+		intVal, err := strconv.Atoi(val)
+		return int64(intVal), err == nil
```

Comment:
Would be good to check error and `return 0, false` in case of error.
I don't think Atoi documents that value will be zero in case of error.

Reply:
Okay, but our generated code doesnt check for value if the returned value is false though. This is a generated snippet. 
```
	if val, ok := getInt64Env(prefix + "MAX_COUNT_PER_ENDPOINT"); ok {
		x.MaxCountPerEndpoint = &wrappers.Int64Value{Value: val}
	} else if x.MaxCountPerEndpoint == nil {
		// when there is no value to set we still prefer to initialize the variable to avoid
		// `nil` checks in the consumers.
		x.MaxCountPerEndpoint = new(wrappers.Int64Value)
		if defaultValues != nil && defaultValues.MaxCountPerEndpoint != nil {
			x.MaxCountPerEndpoint = &wrappers.Int64Value{Value: defaultValues.MaxCountPerEndpoint.Value}
		}
	}
```

Reply:
updated the template

Reply:
Sorry, I did not notice it's generated. if this is generated, it's fine to keep it the way it was.